packages/terra-action-header/tests/jest/__snapshots__/ActionHeader.test.jsx.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ActionHeader should render a default action header 1`] = `<div />`;
```

Comment:
These are not meaningful snapashots, likely because these are the IntlProvider divs

Reply:
The snapshots were updated in this [commit](https://github.com/cerner/terra-core/commit/444fde0838557ce58e76c9faee9b854a6de8d62d).